src/main/java/sirius/pasta/tagliatelle/emitter/LoopEmitter.java

```diff
@@ -91,6 +97,20 @@ protected void emitToContext(LocalRenderContext context) throws Exception {
         }
     }
 
+    private Iterable<?> getItems(Object innerExpression) {
+        if (innerExpression instanceof Iterable<?> iterable) {
+            return iterable;
+        }
+        if (innerExpression.getClass().isArray()) {
+            List<Object> items = new ArrayList<>(Array.getLength(innerExpression));
+            for (int i = 0; i < Array.getLength(innerExpression); i++) {
+                items.add(Array.get(innerExpression, i));
+            }
+            return items;
```

Comment:
Arrays.asList() - this wouldn't require a copy....

Reply:
I could not get this to work, unfortunately :(
When I do `Arrays.asList(innerExpression)` I get a list containing an array, because the compiler does not realize that it's in reality an array. Also, I cannot cast the array to its array type, because I don't know the type of the array. If you have a solution for this, I'm happy to change this

Reply:
On a side node: `Arrays.asList` creates a new `ArrayList` in my JDK, thus also creating a copy

Reply:
Maybe you could write an anonymous Iterable backed by the innerExpression array?

Reply:
done